examples/tv-app/linux/include/zcl-strings.h

```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <app/util/af-types.h>
+
+namespace {
+template <size_t BufferLength>
+uint8_t * MakeZclCharString(uint8_t (&zclString)[BufferLength], char (&cString)[BufferLength])
+{
+    static_assert(BufferLength <= 256, "Too long string to fit in ZCL_CHAR_STRING type");
```

Comment:
```suggestion
    static_assert(BufferLength <= 254, "Too long string to fit in ZCL_CHAR_STRING type");
```

Reply:
I just realized this was copy/pasted from `src/app/clusters/basic/basic.cpp`.  We also have similar code in `examples/bridge-app/linux/main.cpp` (see `ToZclCharString`).

Instead of adding yet another copy of this, can we just have a single utility method?

Reply:
Created ZclString.h and addded it to `src/lib/support`